visualdl/logic/sdk_test.cc

```diff
@@ -132,40 +129,6 @@ TEST(Image, add_sample_test) {
   CHECK_EQ(image2read.num_records(), num_steps);
 }
 
-TEST(Image, add_sample_test) {
```

Comment:
So we do not test Image?

Reply:
we do, its duplicated, add_sample_test already defined